### PR TITLE
Homework 1(Updated) + Homework 2 + Homework 3 + Homework 4

### PR DESCRIPTION
**Pull request by Polyakovst**

### CODE REVIEW

**Review by anatoly-devico — COMMENTED** [review]
push cypress.config.js file to github

File: cypress/selectors/bank_account_screen.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const bank_account_screen = {
+    //Locators for elements on the bank accounts screen
```
Comment (anatoly-devico) [inline cypress/selectors/bank_account_screen.js]: useless comment

File: cypress/selectors/main_screen.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const main_screen = {
+    //Locators for elements on the main screen
```
Comment (anatoly-devico) [inline cypress/selectors/main_screen.js]: useless comment

File: cypress/selectors/sign_up_page.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export const sign_up_page = {
+    //Locators for elements on the 'sign up' page
```
Comment (anatoly-devico) [inline cypress/selectors/sign_up_page.js]: useless comment

File: cypress/selectors/transaction_creation_screen.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+export const transaction_creation_screen = {
+    //Locators for elements on the new transaction creation screen
```
Comment (anatoly-devico) [inline cypress/selectors/transaction_creation_screen.js]: useless comment

File: cypress/selectors/transactions_screen.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export const transactions_screen = {
+    //Locators for elements on transactions screen
```
Comment (anatoly-devico) [inline cypress/selectors/transactions_screen.js]: useless comment

File: cypress/support/commands.js [hunk 6]
```diff
@@ -22,4 +22,43 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
```
Comment (anatoly-devico) [inline cypress/support/commands.js]: Remove unused commented lines

**Review by anatoly-devico — CHANGES_REQUESTED** [review]

File: cypress/selectors/bank_account_screen.js (hunk 1 above)
Comment (Polyakovst) [reply cypress/selectors/bank_account_screen.js]: Comment removed

File: cypress/selectors/main_screen.js (hunk 2 above)
Comment (Polyakovst) [reply cypress/selectors/main_screen.js]: Comment removed

File: cypress/selectors/sign_up_page.js (hunk 3 above)
Comment (Polyakovst) [reply cypress/selectors/sign_up_page.js]: Comment removed

File: cypress/selectors/transaction_creation_screen.js (hunk 4 above)
Comment (Polyakovst) [reply cypress/selectors/transaction_creation_screen.js]: Comment removed

File: cypress/selectors/transactions_screen.js (hunk 5 above)
Comment (Polyakovst) [reply cypress/selectors/transactions_screen.js]: Comment removed

File: cypress/support/commands.js (hunk 6 above)
Comment (Polyakovst) [reply cypress/support/commands.js]: All unused comments removed

**Comment by Polyakovst** [comment]
> push cypress.config.js file to github

cypress.config.сjs file pushed to github

**Review by anatoly-devico — APPROVED** [review]
Approved

**Pull request merged**